src/Features/LanguageServer/Protocol/Handler/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -2,15 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 using System;
 using System.Collections.Generic;
 using System.Composition;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Editor;
 using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
```

Comment:
I would love to get rid of this, but we're currently using https://github.com/dotnet/roslyn/blob/98530991561773ac1fbc2c511f40eab211a11790/src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs#L24 today, and in a couple of places in the LSP. Seems like we'll need some kind of abstraction here.